.gitignore

```diff
@@ -1,104 +0,0 @@
-# Byte-compiled / optimized / DLL files
```

Comment:
Why are we deleting this?

Reply:
Usually we don't use GitHub's default on projects. It's caused lots of pain for us in the past because it ignores some surprising things, like `lib/`(!)